tools/check_style_proto

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Let's name this file `check_style_proto.sh` to indicate it's a shell script.

Reply:
+1